admin/links/class-link-columns.php

```diff
@@ -171,10 +171,10 @@ public function add_post_columns( $columns ) {
 		if ( ! is_array( $columns ) ) {
 			return $columns;
 		}
-		$columns[ 'wpseo-' . self::COLUMN_LINKS ] = '<span class="yoast-linked-to yoast-column-header-has-tooltip" data-label="' . esc_attr__( 'Number of internal links in this post. See "Yoast Columns" text in the help tab for more info.', 'wordpress-seo' ) . '"><span class="screen-reader-text">' . __( '# links in post', 'wordpress-seo' ) . '</span></span>';
+		$columns[ 'wpseo-' . self::COLUMN_LINKS ] = '<span class="yoast-linked-to yoast-column-header-has-tooltip" data-tooltip-text="' . esc_attr__( 'Number of internal links in this post. See "Yoast Columns" text in the help tab for more info.', 'wordpress-seo' ) . '"><span class="screen-reader-text">' . __( 'Internal links', 'wordpress-seo' ) . '</span></span>';
```

Comment:
I would choose for `sprintf` in this case. This makes it more readable.

```suggestion
		$columns[ 'wpseo-' . self::COLUMN_LINKS ] = sprintf( 
				'<span class="yoast-linked-to yoast-column-header-has-tooltip" data-tooltip-text="%1$s"><span class="screen-reader-text">%2$s</span></span>',
				esc_attr__( 'Number of internal links in this post. See "Yoast Columns" text in the help tab for more info.', 'wordpress-seo' ),
				esc_html__( 'Internal links', 'wordpress-seo' )
		);
```

I also used `esc_html__` for the second value.

Reply:
Please verify that the right indentation has been used...